master/reference/calicoctl/resources/ippool.md

```diff
@@ -40,10 +41,9 @@ spec:
 | ipipMode | The IPIP mode defining when IPIP will be used. | Always, CrossSubnet, Never | string| `Never` |
 | natOutgoing | When enabled, packets sent from {{site.prodname}} networked containers in this pool to destinations outside of this pool will be masqueraded. | true, false | boolean | `false` |
 | disabled | When set to true, {{site.prodname}} IPAM will not assign addresses from this pool. | true, false | boolean | `false` |
+| nodeSelector | Selects the node that {{site.prodname}} IPAM should assign addresses from this pool to. | | [selector](#node-selector) | all() |
 
-> **Important**: Do not use a custom `blockSize` until **all** Calico components have been updated to a version that 
-> supports it (at least v3.3.0).  Older versions of components do not understand the field so they may corrupt the 
-> IP pool by creating blocks of incorrect size.
+> **Important**: Do not use the newer fields until **all** Calico components have been updated to a version that supports it. Older versions of components that do not understand the field may corrupt the IP pool.
```

Comment:
"the newer fields"

How will the user know which fields are "the newer fields"?  I think we need to be specific here and not assume that the user has any familiarity with Calico.

Reply:
I think we can delete this note since we are looking at the docs for master and v3.5.

Reply:
Yeah, there's been a whole sub-conversation about how to best express this to the user...
Still not sure what to do... cc @emanic @caseydavenport